src/components/common/CustomImageWrapper.cy.js

```diff
@@ -22,21 +22,27 @@ describe('CustomImageWrapper', () => {
       .should('equal', `${data.image_url}`);
   });
 
-  it('Image renders properly', () => {
-    cy.viewport(1440, 700);
-    mount(
-      <CustomImageWrapper
-        imageUrl={data.image_url}
-        timeCreated={data.time_created}
-        likes={20}
-      />,
-    );
-    cy.get('[alt="tree image"]')
-      .should('be.visible')
-      .and(($img) => {
-        expect($img[0].naturalWidth).to.be.greaterThan(0);
-      });
-  });
+  it(
+    'Image renders properly',
+    {
+      defaultCommandTimeout: 180000,
+    },
```

Comment:
this applies a special config to this test only. I tried 1 minute and the test still failed locally so it's 3 minutes for now.